COMETwebapp/Components/ChartComponents/ChartStackedBarSeries.razor

```diff
@@ -74,4 +78,21 @@ Copyright (c) 2022 RHEA System S.A.
     /// </summary>
     [Parameter]
     public string NameSerie2 { get; set; }
+
+    /// <summary>
+    /// Navigate to the page with details for these selected data
+    /// </summary>
+    /// <param name="data">List of the selected data</param>
+    public void AccessDataDetails(List<DataChart> data)
+    {
+        string iids = "";
+        data.ForEach(d =>
+        {
+            if (d.Parameters.Any())
+            {
+                d.Parameters.ForEach(p => iids += $"|{p.Iid.ToString()}");
+            }
+        });
+        NavigationManager.NavigateTo($"/ModelDashboard/Details?iids={iids}");
```

Comment:
as long as the amount of parameters is limited, this will work, but my expectation is that for medium to large models this may become at least 100 items long, and then i doubt this will work. And, 100 is not even that many. In a team with 10 domains-of-expertise it only takes 10 parameters each.. So, i think once we have verified that this is a useful feature to have we have to figure out a different way to pass that data